Assets/Vimeo/Scripts/Services/VimeoApi.cs

```diff
@@ -335,11 +339,13 @@ IEnumerator Request(string api_path)
                     if (request.responseCode == 401) {
                         Debug.LogError("[VimeoApi] 401 Unauthorized request.");
                     }
-                    else {
-                        JSONNode json = JSON.Parse(request.downloadHandler.text);
-                        Debug.LogError("[VimeoApi] " + request.responseCode + " " + json["error"]);
+                    if (OnError != null && OnNetworkError != null) {
```

Comment:
Decouple this so that you can bind into either event if you want.

Reply:
https://github.com/vimeo/vimeo-unity-sdk/pull/28/commits/056dc50026b3e527bf7f666ccaa9f5a89e05416f